src/Symfony/Component/PropertyInfo/Tests/Extractor/ReflectionExtractorTest.php

```diff
@@ -261,4 +261,28 @@ public function testSingularize()
         $this->assertTrue($this->extractor->isWritable(AdderRemoverDummy::class, 'feet'));
         $this->assertEquals(['analyses', 'feet'], $this->extractor->getProperties(AdderRemoverDummy::class));
     }
+
+    /**
+     * @dataProvider constructorTypesProvider
+     */
+    public function testExtractTypeConstructor(string $class, string $property, array $type = null)
```

Comment:
```suggestion
    public function testExtractTypeConstructor(string $class, string $property, array $type)
```

you alway provide a third argument, so no need for a default value here

Reply:
actually, seems the 3rd argument is needed - as I provide null for the last test case - which isn't an array.

Reply:
But can the type variable be null in the real world or only in a test scenario?
I am not that deep into it but maybe removing the testcase or using „?array“ to allow null but not set it if nothing is set?